src/api/File.js

```diff
@@ -92,32 +91,35 @@ class File extends Item {
         const infoUrl = getProp(representation, 'info.url');
 
         const numOfTries = 3;
-        const initialTimeout = 3000;
-        const backoffFactor = 2;
+        const initialTimeout = 500;
```

Comment:
Perhaps it would be worth checking with conversion to team to see what they recommend for a polling interval that would make sense?